### PR TITLE
Added support notification payload

### PR DESCRIPTION
**Pull request by VasayXTX**
Hi! Thanks for lib! I need notification payload in my project, so please approve this pr.

### CODE REVIEW

File: library/CodeMonkeysRu/GCM/Sender.php [hunk 1]
```diff
@@ -165,4 +163,24 @@ private function formMessageData(Message $message)
...
+    /**
+     * Validate size of json representation of passed payload
+     *
+     * @param array $rawData
+     * @param string $fieldName
+     * @param int $maxSize
+     * @throws \CodeMonkeysRu\GCM\Exception
+     * @return void
+     */
+    private static function validatePayloadSize(array $rawData, $fieldName, $maxSize)
```
Comment (vladimirsavenkov) [inline library/CodeMonkeysRu/GCM/Sender.php]: Can you please remove "static" signature?

**Comment by vladimirsavenkov** [comment]
Thx for PR! Can you please fix one small issue (see comments above)?

**Comment by VasayXTX** [comment]
Ok, I've done it. Also I've added using notification payload in the readme.md.

**Comment by iVariable** [comment]
Thx!

**Pull request merged**